src/components/modal/event-community/ModalViewDetailEvent.jsx

```diff
@@ -0,0 +1,229 @@
+          right={"1.5rem"}
+          top={"1.5rem"}
+          _hover={{ bg: "transparent", color: "#333333" }}
+          _focus={{ boxShadow: "none" }}
+        />
+        <ModalHeader p={0}>
+          <Heading
+            as="h3"
+            fontSize={"2xl"}
+            fontWeight={"medium"}
+            letterSpacing={"tight"}
+            color={"#000000"}
+          >
+            Detail Komunitas
+          </Heading>
+        </ModalHeader>
+        <ModalBody
+          display={"grid"}
+          gridTemplateColumns={"0.5fr 1.5fr"}
+          gap={6}
+          p={0}
+        >
+          <Flex
+            position={"relative"}
+            aspectRatio={"4/3"}
+            borderRadius={"lg"}
+            flexDirection={"column"}
+            gap={"1rem"}
+          >
+            <Text color={"#828282"} fontSize={"lg"}>
+              Gambar Event
+            </Text>
+            <Image
+              src={data?.image}
+              alt="community image"
+              aspectRatio={"4/3"}
+              objectFit={"cover"}
+              borderRadius={"lg"}
+            />
+          </Flex>
+
+          <Grid
+            templateRows={"auto auto auto"}
+            templateColumns={"1fr 1fr 1fr"}
+            gap={6}
+          >
+            <GridItem colSpan={"3"}>
+              <Flex flexDirection={"column"} gap={"0.5rem"}>
+                <Text color={"#828282"} fontSize={"lg"}>
+                  Nama Event
+                </Text>
+                <Heading color={"#3BA639"} fontSize={"4xl"} fontWeight={"bold"}>
+                  {data?.name}
+                </Heading>
+              </Flex>
+            </GridItem>
+            <GridItem
+              as={Flex}
+              colSpan={"3"}
+              flexDirection={"column"}
+              gap={"0.5rem"}
+            >
+              <Text color={"#828282"} fontSize={"lg"}>
+                Deskripsi Event
+              </Text>
+              <Text color={"#333333"} fontSize={"sm"} textAlign={"justify"}>
+                {data?.descriptions}
+              </Text>
+            </GridItem>
+
+            <GridItem
+              as={Flex}
+              colSpan={"3"}
+              flexDirection={"column"}
+              gap={"0.5rem"}
+            >
+              <Text color={"#828282"} fontSize={"lg"}>
+                Lokasi
+              </Text>
+              <Text color={"#333333"} fontSize={"sm"} textAlign={"justify"}>
+                {data?.location}
+              </Text>
+            </GridItem>
+
+            {/* link google maps */}
+            <GridItem
+              as={Flex}
+              colSpan={"3"}
+              flexDirection={"column"}
+              gap={"0.5rem"}
+            >
+              <Text color={"#828282"} fontSize={"lg"}>
+                Link Google Maps Lokasi Event
+              </Text>
+              <Text color={"#333333"} fontSize={"sm"} textAlign={"justify"}>
+                {data?.gMaps}
```

Comment:
![image](https://github.com/RecyThing/RecyThing-Web/assets/98959851/3b8ab978-7186-4752-b92c-2edcddaa35c7)

maps sama form kurang aksi buat salin datanya, bisa pake Clipboard API kali

https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript